### DISCUSSION TITLE
Welcome to PythonRobotics Discussions!

### DISCUSSION THREAD

**Discussion by AtsushiSakai** [AtsushiSakai/PythonRobotics#762] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by BEgilson** [comment]
Hey,

My name is Benóný and I have been learning and building with Python for a few years.

I started my coding journey with Html and Css and I'm trying to expand my skills in all directions

I love learning and developing my skills.

**Comment by Sriram-TSci** [comment]
By any chance can you add comments for your code on pure pursuit algorithm so that we can understand it better.
thank you

**Comment by PKolew** [comment]
Hi, 

I'm Pavel. I'm a newbie in Python learning it since January last year 🤙. Wanna go big in the programming's world 💪

**Comment by SCUTBob** [comment]
I'm confused about the model used by LQR. I did not get the same model with direct linearization of the single bike model, does anyone provide the corresponding derivation or paper?

**Comment by nguyenhoangtan4409** [comment]
Hello,
I wonder in the PathTracking/cgmres_nmpc section, can I change the destination goal?
If yes, how can I modify it?

**Comment by tushar0626** [comment]
Hi, 
I am doing a project and first of all I would like to thank @AtsushiSakai for your Frenet Optimal Trajectory Generation. 

**Doubt:**
In the code ( Frenet optimal trajectory generation ) - In the if condition if np.hypot(path.x[1] - tx[-1], path.y[1] - ty[-1]) <= 1.0:  
                print("GOAL reached"), why I am not able to put 0.0 instead of 1.0 m, 
Is there any fix for this so that my object exactly reaches the specified point? if I give 0.0 instead of 1.0 in the if condition, I am getting an error message as  

fp.yaw.append(fp.yaw[0])
                  ~~~~~~^^^
IndexError: list index out of range   

In calc_global_path function. 

I would really appreciate if there is any possible solution for this. 

Thanks,

**Reply by huagkcjeim** [reply]
what
